spaces/bti/index.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "bti.finance",
+  "network": "1",
+  "symbol": "bti",
+  "skin": "example",
```

You will need to remove skin field and add domain field to mae your domain working.
```suggestion
  "domain": "gov.bti.finance",
```